### PR TITLE
added secound version CSS

### PR DESCRIPTION
**Pull request by toppy007**
Have made a start on the CSS styling for the whole website.

Maybe worth downloading and running the brach to view the changes. 

Main change is the dropdown select, where Ive imported react select to help.

Tests all passing (but have had to remove one from the forms.cy), (added a interception to mock the API in cypress for the Results Page)

Have also added a few background images.

### CODE REVIEW

**Review by umutbaykan — CHANGES_REQUESTED** [review]
no objections to the code, but can we link the jpeg images instead of putting them in our repo? the current repo size grew to around 30-35mb, which is quite a lot for a webpage

File: frontend/src/components/app/App.js [hunk 1]
```diff
@@ -10,12 +10,14 @@ import ResultPage from '../result-page/ResultPage';
...
   return (
-    <div>
-      <h1 className="title">AI-Tastic Tales</h1>
-      <Routes>
-        <Route path="/" element={<FormContainer navigate={ useNavigate() }/>} />
-        <Route path="/results" element={<ResultPage navigate={ useNavigate() }/>} />
-      </Routes>
+    <div className="background-image-container">
+      <div>
+        {/* <h1 className="title">AT</h1> */}
```
Comment (natashabuckham) [inline frontend/src/components/app/App.js:15]: Should this line be deleted?

File: frontend/cypress/fixtures/images.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "imgUrl": "image-url"
```
Comment (natashabuckham) [inline frontend/cypress/fixtures/images.json:2]: Let's change the string to an image url as Umut suggested :)

**Review by natashabuckham — CHANGES_REQUESTED** [review]

File: frontend/src/components/app/App.js (hunk 1 above)
Comment (toppy007) [reply frontend/src/components/app/App.js:15]: will delete thanks

**Comment by toppy007** [comment]
will review your second comment natasha when i start styling the resultpage

**Review by umutbaykan — APPROVED** [review]

**Review by natashabuckham — APPROVED** [review]

**Pull request merged**